indra/llcharacter/llik.cpp

```diff
@@ -525,7 +599,7 @@ LLIK::ElbowConstraint::ElbowConstraint(const LLVector3& forward_axis, const LLVe
     mForward.normalize();
     mPivotAxis = mForward % (pivot_axis % mForward);
     mPivotAxis.normalize();
-    mLeft = mPivotAxis % mForward;
+    mPivotXForward = mPivotAxis % mForward;
```

Comment:
The name `ElbowConstraint::mLeft` was unclear so I changed it to something more meaningful.